pkg/volume/scaleio/sio_volume.go

```diff
@@ -248,12 +248,12 @@ func (v *sioVolume) Provision() (*api.PersistentVolume, error) {
 	// setup volume attrributes
 	name := v.generateVolName()
 	capacity := v.options.PVC.Spec.Resources.Requests[api.ResourceName(api.ResourceStorage)]
-	if capacity.Value() == 0 {
```

Comment:
I liked this code, refusing wrong PVCs is IMO better than defaulting to magic 8GiB.